Gemfile

```diff
@@ -14,8 +14,8 @@ group :test do
   gem 'capybara'
   gem 'launchy'
   gem 'poltergeist', require: false
-  gem 'selenium-webdriver', require: false
   gem 'database_cleaner'
+  gem "rubycritic", require: false
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols. [:ref](https://github.com/bbatsov/ruby-style-guide#consistent-string-literals)

<!-- Comment by [SideCI](https://sideci.com) -->